### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (label)

### ISSUE THREAD

**Issue by hash-logic** [hash-logic/issue#6]
**Issue**: `Elements must have sufficient color contrast` ([`color-contrast`](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))

**Target application**: [RFQDashboard · Custom Portal](https://mymatterssit.microsoftcrmportals.com/RFQDashboard/)

**Element path**: tr:nth-child(1) > td[aria-label="Status\ RFQ\ in\ Progress"][data-th="Status"] > label

**Snippet**:

    <label id="status" style="background-color: rgb(255,140,0) ;padding-left:10px; padding-right:10px; padding-top:5px; padding-bottom:5px; font-weight:normal !important; color: white !important;width:180px; text-align:center;">RFQ in Progress</label>

**How to fix**:

    Fix any of the following:
      Element has insufficient color contrast of 2.33 (foreground color: #ffffff, background color: #ff8c00, font size: 9.8pt, font weight: normal). Expected contrast ratio of 4.5:1

**Environment**:
Chrome version 73.0.3683.86

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
